### PR TITLE
#37  Details of SuperOvers and #35 Most Wicket in first Over by a Team

### PR DESCRIPTION
**Pull request by kavania2002**

### CODE REVIEW

**Comment by sakshamj74** [comment]
@kavania2002 Plot graph of what you analyzed and your outcomes from that analysis.
These tables are difficult to interpret. And move your analysis code to a separate file.

**Comment by kavania2002** [comment]
Ohk Fine

**Comment by kavania2002** [comment]
I have analysed the number of runs given by a bowler in a Super Over
Please Review

**Comment by kavania2002** [comment]
SuperOver

**Comment by kavania2002** [comment]
The photo of Most Wickets by a team in First Over
MostWickertFirstOver

File: Deliveries.ipynb [hunk 1]
```diff
@@ -1454,10 +1454,388 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 4,
    "metadata": {},
-   "outputs": [],
-   "source": []
+   "outputs": [
```
Comment (sakshamj74) [inline Deliveries.ipynb]: I think this code is not required here because all of it has been moved to other files.

**Review by sakshamj74 — CHANGES_REQUESTED** [review]

File: Deliveries.ipynb (hunk 1 above)
Comment (kavania2002) [reply Deliveries.ipynb]: Actually I still can't understand. I am not getting you.

Comment (kavania2002) [reply Deliveries.ipynb]: Ohk now I get it, yes Yes

**Comment by kavania2002** [comment]
I had to create new pull request. Sorry for the inconvenience.

**Pull request closed without merging**